ksqldb-engine/src/test/java/io/confluent/ksql/integration/SecureIntegrationTest.java

```diff
@@ -307,6 +309,7 @@ public void shouldClassifyMissingSourceTopicExceptionAsUserErrorSharedRuntimes()
   }
 
   @Test
+  @Ignore
```

Comment:
This and the one below do get classified as user errrors and do happen. But as the error occurs in streams and happens after the topology gets added the error is not thrown up to the statement response. The error is in the error queue for the query though